vendor/build.gradle

```diff
@@ -0,0 +1,168 @@
+project(':vendor') {
+  apply plugin: 'cpp'
+
+  model {
+    components {
+      // We are building protoc and protobuf from source for maximum compatibility
+      // across platform. Please note that we are using a very minimilistic
+      // port of the build to Gradle.
```

Comment:
I extracted the information by hand from the Bazel build script. Since I didn't want to make the Bazel tools part of the environment (as it wasn't previously) I opted to include the build logic inside Gradle instead since it's perfectly capable to build the Protobuf library. In the future, we could extract the information on the fly which will make the script more flexible when changing the submodule tag.